docs/steps.txt

```diff
@@ -4,20 +4,55 @@ ________________
 
 Task 1:
 
+First of all I created a pom file dependency file that has all of the dependency i need (web app, mongo database , health checker)
+at my CounterServlet.java controller file a add a annotations to till set my application as spring boot application.
+
+I started with my hello world web app then how to get the count of the visitors for my web app then get the ip address.
+
+Now I start to config the mongo database to get requests but i could not do that by my self honestly I asked a friend
```

Comment:
if you don't feel comfortable with Java, you can use easier language like Python or NodeJS.